gluten-flink/planner/src/main/java/org/apache/gluten/rexnode/functions/RexCallConverterFactory.java

```diff
@@ -56,10 +56,22 @@ public class RexCallConverterFactory {
                   () -> new BasicArithmeticOperatorRexCallConverter("equalto"),
                   () -> new StringCompareRexCallConverter("equalto"),
                   () -> new StringNumberCompareRexCallConverter("equalto"))),
+          Map.entry("/", Arrays.asList(() -> new DecimalRexCallConverters("divide"))),
```

Comment:
Does `divide` needs `BasicArithmeticOperator`? it seems the  division for `basic numuber` is much more faster than `decimal` divide.